docs/extensibility/annotations.md

```diff
@@ -0,0 +1,55 @@
+# Annotations
```

Comment:
If you want to show this page somewhere in the documentation, you need to update the root-manifest.json and run the docs generation command.

Reply:
> If you want to show this page somewhere in the documentation, you need to update the root-manifest.json and run the docs generation command.

Added in ebf62c90974ec47fb56ef02ce067165c516adddd.